src/main/java/sysc4806/graduateAdmissions/model/Application.java

```diff
@@ -0,0 +1,57 @@
+package sysc4806.graduateAdmissions.model;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+/**
+ * An object that describes an application to a degree in our system. An application has
+ * an applicant, a department, degree, and professors that they are applying to study under,
+ * a status that describes what stage the application is in, and any information inputted by the
+ * applicant in the application forms.
+ *
+ * @author madelynkrasnay
+ */
+@Entity
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Application {
+
+    //generated Primary key
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @EqualsAndHashCode.Exclude
+    private long id;
+
+    //The owner of the application
+    //TODO: make a user when user object is created
+    private String applicant;
+
+    //term that the application is for
+    private Term term;
+
```

Comment:
Since Term is an Entity, we need to tell spring that there is a @ OneToOne relationship between terms and applications. Add this annotation to Term to fix the build issue.